controls/settings/alpha/update.go

```diff
@@ -0,0 +1,68 @@
+package alpha
+
+import (
+	"github.com/charmbracelet/bubbles/key"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/Zebbeni/ansizalizer/event"
+)
+
+type Direction int
+
+const (
+	Up Direction = iota
+	Left
+	Right
+)
+
+var (
+	navMap = map[Direction]map[State]State{
+		Right: {AlphaYes: AlphaNo},
+		Left:  {AlphaNo: AlphaYes},
+	}
+)
+
+func (m Model) handleEsc() (Model, tea.Cmd) {
+	m.ShouldUnfocus = true
+	m.IsActive = false
+	return m, nil
+}
+
+func (m Model) handleNav(msg tea.KeyMsg) (Model, tea.Cmd) {
+	switch {
+	case key.Matches(msg, event.KeyMap.Right):
+		if next, hasNext := navMap[Right][m.focus]; hasNext {
+			return m.setFocus(next)
+		}
+	case key.Matches(msg, event.KeyMap.Left):
+		if next, hasNext := navMap[Left][m.focus]; hasNext {
+			return m.setFocus(next)
+		}
+	case key.Matches(msg, event.KeyMap.Up):
+		m.ShouldUnfocus = true
+	}
+	return m, nil
+}
+
+func (m Model) setFocus(focus State) (Model, tea.Cmd) {
+	if focus == AlphaYes {
+		m.useAlpha = true
+		m.focus = AlphaYes
+	}
+
+	if focus == AlphaNo {
+		m.useAlpha = false
+		m.focus = AlphaNo
+	}
+
+	return m, nil
+}
+
+func (m Model) handleEnter() (Model, tea.Cmd) {
+	switch m.focus {
+	case AlphaYes:
+		m.useAlpha = true
+	case AlphaNo:
+		m.useAlpha = false
+	}
+	return m, event.StartRenderToViewCmd
```

Comment:
For some reason when I test this branch, I'm not seeing my image update when I go back and forth hitting "Enter" on Alpha Channel: Yes and Alpha Channel: No. It looks like the `useAlpha` attribute is updated, but I don't see a re-render until I toggle another setting somewhere. Is the `StartRenderToViewCmd` getting intercepted and replaced with nil somewhere...?

Reply:
This should be fixed :)